ee/server/service/mdm.go

```diff
@@ -560,53 +560,30 @@ func (svc *Service) InitiateMDMAppleSSOCallback(ctx context.Context, auth fleet.
 	return appConfig.ServerSettings.ServerURL + "/mdm/sso/callback?" + q.Encode(), nil
 }
 
-func (svc *Service) mdmAppleSyncDEPProfile(ctx context.Context) error {
+func (svc *Service) mdmAppleSyncDEPProfiles(ctx context.Context) error {
+	// TODO(mna): all profiles must be updated: this gets called when the ServerURL or MDM
+	// SSO got modified. And then all devices part of the profile's team must be re-assigned
+	// the updated profile. Enqueue a worker job to take care of this.
```

Comment:
@roperzh So the plan would be for this PR to merge like this (feature-wise, I'll address any review comments of course), and in the next PR I'll add the worker messages and processing.